src/pages/EnspirePage/components/Register.js

```diff
@@ -7,7 +7,53 @@ class Register extends Component {
         //write your css in src/scss/apps/enspire/components/_register.scss
         return(
             <div className="w-100 h-100 enspire-register">
-                This is the register page
+
+                {/* Title */}
+                <div className="enspire-register__head">
+                    <b>Register for Enspire!</b>
+                </div>
+
+                {/* 1x2 grid with words on left and picture on right */}
+                <div class="container">
+                    <div class="row">
+                        {/* For the words on the left */}
+                        <div class="col">
```

Comment:
```suggestion
                        <div class="col-md-6">
```

Using `col` isn't making the columns stack on small screen sizes. To see what I mean by stack - look at the landing page of tesc.ucsd.edu/decaf : On small screen sizes the 2 columns stack on top of each other instead of splitting 50% of the screen on an already tiny screen (this is what is happening now).